src/main/java/net/datafaker/providers/base/BaseFaker.java

```diff
@@ -400,4 +400,9 @@ public final <B extends ProviderRegistration> B getFaker() {
     public static Method getMethod(AbstractProvider<?> ap, String methodName) {
         return ap == null ? null : ObjectMethods.getMethodByName(ap, methodName);
     }
+
+    @Override
+    public String toString() {
```

Comment:
isn't this the default toString?

Reply:
Almost. Except that it's shorter due to `getSimpleName` instead of `getName`. 
I would like to make it even more shorter, but it doesn't have any human-readable fields...

Reply:
but why do you need it?